src/widgets/openmediavault/OpenMediaVaultFileSystem.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Group, Paper, Progress, Text } from '@mantine/core';
+import { IconServer } from '@tabler/icons-react';
+
+const OpenMediaVaultFileSystem = ({ fileSystem }: any) => {
+  function formatBytes(bytes: number): string {
```

Comment:
I think there's already a function for this in our tools:
import {humanFileSize} from '~/tools/humanFileSize'

Reply:
great I use it.